### PR TITLE
Create lorawan-spec.md

### PR DESCRIPTION
**Pull request by rish-c**
Redirecting to LoRaWAN specs

### CODE REVIEW

File: _content/lorawan/lorawan_specs.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+The LoRaWAN™ specification documents are developed and maintained by the LoRa Alliance: an open association of collaborating members. To fully define the LoRaWAN protocol, and to ensure interoperability among devices and networks, the LoRa Alliance develops and maintains the following documents:
+
+* Core LoRaWAN Specification
+* Regional Parameters
+* Certification Program
+* Back-End Interfaces
+
+While these specifications define the technical implementation, they do not define any commercial model or type of deployment (public, shared, private, enterprise) and so offer the industry the freedom to innovate and differentiate in how a LoRaWAN system is implemented.
+
+The specification defines the and so provides seamless interoperability between manufacturers, as demonstrated via the device certification program.
```
Comment (egourlao) [inline _content/lorawan/lorawan_specs.md]: > The specification defines the and

Missing word?

**Review by egourlao — CHANGES_REQUESTED** [review]

File: _content/lorawan/lorawan_specs.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+---
+title: LoRaWAN Specifications
```
Comment (KrishnaIyer) [inline _content/lorawan/lorawan_specs.md]: Specifications -> Specification (Collective Noun)

File: _content/lorawan/lorawan_specs.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+---
+
+# LoRaWAN Specifications
```
Comment (KrishnaIyer) [inline _content/lorawan/lorawan_specs.md]: Specifications -> Specification (Collective Noun)

File: _content/lorawan/lorawan_specs.md (hunk 1 above)
Comment (KrishnaIyer) [reply _content/lorawan/lorawan_specs.md]: Try this for brevity..
defines the and so provides -> enables

File: _content/lorawan/lorawan_specs.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+* Certification Program
+* Back-End Interfaces
+
+While these specifications define the technical implementation, they do not define any commercial model or type of deployment (public, shared, private, enterprise) and so offer the industry the freedom to innovate and differentiate in how a LoRaWAN system is implemented.
+
+The specification defines the and so provides seamless interoperability between manufacturers, as demonstrated via the device certification program.
+
+![file](https://ttnstaticfile.blob.core.windows.net/media/md_editor/image-1526028762688.large.png)
+
+You can find more information on the LoRaWAN specifactions from the link [here](https://lora-alliance.org/lorawan-for-developers).
```
Comment (johanstokking) [inline _content/lorawan/lorawan_specs.md]: Typo

File: _content/lorawan/lorawan_specs.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
+---
+title: LoRaWAN Specifications
+---
+
+# LoRaWAN Specifications
+
+The LoRaWAN™ specification documents are developed and maintained by the LoRa Alliance: an open association of collaborating members. To fully define the LoRaWAN protocol, and to ensure interoperability among devices and networks, the LoRa Alliance develops and maintains the following documents:
```
Comment (johanstokking) [inline _content/lorawan/lorawan_specs.md]: This is copied straight from their website, right? I'm not sure if we should do that.

**Review by johanstokking — CHANGES_REQUESTED** [review]

File: _content/lorawan/lorawan_specs.md [hunk 6]
```diff
@@ -13,7 +13,7 @@ The LoRaWAN™ specification documents are developed and maintained by the LoRa
 
 While these specifications define the technical implementation, they do not define any commercial model or type of deployment (public, shared, private, enterprise) and so offer the industry the freedom to innovate and differentiate in how a LoRaWAN system is implemented.
 
-The specification defines the and so provides seamless interoperability between manufacturers, as demonstrated via the device certification program.
+The specification defines provides seamless interoperability between manufacturers, as demonstrated via the device certification program.
```
Comment (egourlao) [inline _content/lorawan/lorawan_specs.md]: > defines provides

Still a typo, no? There are two verbs in the sentence

File: _content/lorawan/lorawan_specs.md [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+# LoRaWAN Specification
+
+The LoRaWAN™ specification documents are developed and maintained by the LoRa Alliance: an open association of collaborating members. To fully define the LoRaWAN protocol, and to ensure interoperability among devices and networks, the LoRa Alliance develops and maintains the following documents:
+
+* Core LoRaWAN Specification
+* Regional Parameters
+* Certification Program
+* Back-End Interfaces
+
+While these specifications define the technical implementation, they do not define any commercial model or type of deployment (public, shared, private, enterprise) and so offer the industry the freedom to innovate and differentiate in how a LoRaWAN system is implemented.
```
Comment (johanstokking) [inline _content/lorawan/lorawan_specs.md:14]: This is too much a copy paste

File: _content/lorawan/lorawan_specs.md [hunk 8]
```diff
@@ -0,0 +1,20 @@
...
+
+The specification defines and provides seamless interoperability between manufacturers, as demonstrated via the device certification program.
+
+![file](https://ttnstaticfile.blob.core.windows.net/media/md_editor/image-1526028762688.large.png)
+
+You can find more information on the LoRaWAN specifactions from the link [here](https://lora-alliance.org/lorawan-for-developers).
```
Comment (johanstokking) [inline _content/lorawan/lorawan_specs.md:20]: Typo

**Review by johanstokking — CHANGES_REQUESTED** [review]

File: _content/lorawan/lorawan_specs.md [hunk 9]
```diff
@@ -0,0 +1,20 @@
...
+* Core LoRaWAN Specification
+* Regional Parameters
+* Certification Program
+* Back-End Interfaces
+
+While these specifications define the technical implementation, they do not define any commercial model or type of deployment (public, shared, private, enterprise) and so offer the industry the freedom to innovate and differentiate in how a LoRaWAN system is implemented.
+
+The specification defines and provides seamless interoperability between manufacturers, as demonstrated via the device certification program.
+
+![file](https://ttnstaticfile.blob.core.windows.net/media/md_editor/image-1526028762688.large.png)
```
Comment (htdvisser) [inline _content/lorawan/lorawan_specs.md:18]: `file` is not a proper alt description

**Comment by johanstokking** [comment]
@rish-c please reopen when comments are addressed, closing for inactivity now.

**Pull request closed without merging**